content/en/logs/guide/control-sensitive-logs-data.md

```diff
@@ -35,90 +41,101 @@ Controlling all of your data can be challenging, especially on a large and highl
+If certain kinds of sensitive data are prohibited from leaving your environment and being ingested into Datadog, then add scrubbing rules at source collection.
+
+If you have way to change the loggers themselves, Datadog provides you with solutions to prevent compliance-sensitive data from being sent outside of your platform when using the [Datadog Agent][13] for Log Collection:
 
-* [Scrub sensitive data][12] from your logs before you send them to Datadog.
-* Alternatively, use [Autodiscovery][13] to add fine-grained controls for containers log collection.
+* [Scrub sensitive data][14] from your logs before you send them to Datadog.
+* Alternatively, use [Autodiscovery][15] to add fine-grained controls for containers log collection.
 
-Similar scrubbing capabilities exist for the [Serverless Forwarder][14].
+Similar scrubbing capabilities exist for the [Serverless Forwarder][16].
 
-## Handle data already sent to Datadog
+## Handle data already sent to and indexed in Datadog
 
 Take the following steps according to your compliance requirements. You might not need to complete all steps.
 
 ### Make sensitive logs un-queryable in Datadog (until they age-out)
 
 This step makes logs with sensitive data, both logs that already sent and logs that might keep flowing in, not queryable in Datadog (Explorer, Dashboards, and Livetail).
 
-Use the [Data Access configuration page][15] and a sensitive outline query to define a [restriction][16] that applies to everyone in your organization. For example, the query mentioned above: `version:x.y.z source:python status:debug`.
+Use the [Data Access configuration page][17] and a sensitive outline query to define a [restriction][18] that applies to everyone in your organization. For example, the query mentioned above: `version:x.y.z source:python status:debug`.
 
 **Note:** Using **NOT** in the sensitive outline query restricts users from seeing anything BUT matching logs.
 
 {{< img src="logs/guide/sensitive/sensitive_data_access.png" alt="Sensitive Data Access" style="width:80%;" >}}
 
 ### Patch your archives
 
-If you have to patch your archives to remove sensitive data, refer to the [format of archives][17] generated by Datadog documentation.
+If you have to patch your archives to remove sensitive data, refer to the [format of archives][19] generated by Datadog documentation.
 
 ## Support
 
-If you have a specific compliance questions or need help, contact Datadog [support][18]. When you contact support, it is helpful for you to have the following information available:
+If you have a specific compliance questions or need help, contact Datadog [support][20]. When you contact support, it is helpful for you to have the following information available:
 
 * The sensitive outline query or anything that could define sensitive data such as a timerange, a service, or an environment.
+* Whether you use [Sensitive Data Scanner][21].
```

Comment:
We should maintain consistency and keep linking this the way we have in the rest of the doc. Add the blog link to further reading instead.

```suggestion
* Whether you use [Sensitive data scanner][2].
```